### DISCUSSION TITLE
Difficulty sorting with mods not working

### DISCUSSION THREAD

**Discussion by thybomb** [ppy/osu#28957] [Ideas] [answered]
On stable, when I have maps sorted by difficulty, turn on a mod, and then go to main menu and back to song select, the maps are now sorted by the star rating after applying those mods, but on lazer it still sorts by the no mod star rating, can this be changed to match stable?

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/12628.
